rubocop-base.yml

```diff
@@ -485,9 +485,10 @@ Layout/ExtraSpacing:
   Description: Do not use unnecessary spacing.
   Enabled: true
 Style/MethodCallWithArgsParentheses:
+  Enabled: false
+Style/MethodCallWithoutArgsParentheses:
   Enabled: true
-  EnforcedStyle: require_parentheses
-  IgnoreMacros: true
+  AllowedMethods: []
```

Comment:
Con la configuración actual:

- `Style/MethodCallWithArgsParentheses` está deshabilitada, así que RuboCop no exige ni prohíbe paréntesis cuando un método recibe argumentos. Podés escribir foo 1, 2 o foo(1, 2) sin que marque nada. Esto replica el comportamiento de Standard y evita choques con DSLs (RSpec, Rails helpers, etc.).
- `Style/MethodCallWithoutArgsParentheses` está habilitada con `AllowedMethods: []`, por lo que RuboCop exige que las llamadas sin argumentos no lleven paréntesis. Es decir, foo() dispara una ofensa y sugiere foo. Esto favorece la sintaxis “limpia” para métodos sin parámetros.